week-2-practise/level-2/4-slice.js

```diff
@@ -5,21 +5,29 @@
 // - remove the item at the specified index
 
 function remove(arr, index) {
-  return; // complete this statement
+  var newArr = arr.slice(0, index);
+  var newArr2 = arr.slice(index, arr.lenght);
+  newarr3 = newArr.concat(newArr2);
+
+  return console.log(arr.length);
+  return console.log(newArr2);
+  return console.log(newArr);
 }
 
 /* 
   DO NOT EDIT BELOW THIS LINE
   --------------------------- */
-var numbers = [1, 2, 3];
+var numbers = [1, 2, 3, 4, 5, 6];
 var names = ["Irina", "Ashleigh", "Mozafar"];
 
-var newNumbers = remove(numbers, 2);
+var newNumbers = remove(numbers, 4);
 var newNames = remove(names, 1);
-
+/*
 console.log(newNumbers);
 console.log(newNames);
-
+console.log(numbers);
+console.log(names);
+*/
```

Comment:
Oh, I see now what you were saying about the `slice` exercise. First, I'd recommend removing all the changes that you made after this line:

```
 DO NOT EDIT BELOW THIS LINE
```

Then, take a look at the description for the `end` argument for `array.slice()` in the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice). Can you think of a way to use the `end` argument of `array.slice()` so that you only remove a single item?